remote-api/src/main/java/org/alfresco/rest/api/impl/CategoriesImpl.java

```diff
@@ -341,6 +371,21 @@ private Collection<NodeRef> mergeCategories(final Serializable currentCategories
         return allCategories;
     }
 
+    /**
+     * Remove specified category from present categories.
+     * @param currentCategories already present categories.
+     * @param categoryToRemove category that should be removed.
+     * @return updated category list.
+     */
+    private Collection<NodeRef> removeCategory(final Serializable currentCategories, final NodeRef categoryToRemove)
```

Comment:
I would prefer the `Collection<NodeRef>` to be passed in for `currentCategories` or the noderef to be passed in instead. The fact that the properties is a map to Serializable is a slightly ugly implementation detail and I would try to keep that out of the method contract.

This is pretty subjective - I just wanted to discuss it; I don't really mind if you leave it as it is.